schemas/lib.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: lib
+  title: Shared Library
```

Comment:
Please rename to "Library", as the sorting [here](http://ecs_679.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-field-reference.html) will be confusing. Same argument as the for Registry :-)

Reply:
Renaming lib -> library makes sense. I was just trying to find a way to add a little bit of clarity to the title